recOrder/scripts/cli.py

```diff
@@ -1,40 +1,106 @@
 import click
+import napari
+import numpy as np
 import sys
-import os
+
+# Workaround waveorder #97
+# Create napari Viewer before other imports
+v = napari.Viewer()
+v.close()
```

Comment:
Will these lines take the first CLI command ~10s to respond since a QApp has to be spun up and torn down?